HOWTO-fil.md

```diff
@@ -0,0 +1,17 @@
+Basahin ito sa ibang mga wika: [English](HOWTO.md), [Français](HOWTO-fr.md), [Español](HOWTO-es.md), **Filipino**, [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md), [Deutsch](HOWTO-de.md), [Bahasa Indonesia](HOWTO-id.md), [Tiếng Việt](HOWTO-vi.md), [عربي](HOWTO-ar.md).
+
+Maligayang pagdating sa Free-Programming-Books! Tinatanggap namin ang mga bagong kontribyutor; kahit na ang mga gumagawa ng kanilang pinakaunang pull request sa Github. Kung isa ka sa mga iyon, narito ang ilang mapagkukunan na maaaring makatulong:
+
+* [About Pull Requests](https://help.github.com/articles/about-pull-requests/)
+* [Creating a pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
+* [Github Hello World](https://guides.github.com/activities/hello-world/)
+* [Youtube - Github Tutorial For Beginners](https://www.youtube.com/watch?v=0fKg7e37bQE)
+* [Youtube - How To Fork A GitHub Repo and Submit A Pull Request](https://www.youtube.com/watch?v=G1I3HF4YWEw)
+* [Youtube - Markdown Crash Course](https://www.youtube.com/watch?v=HUBNt18RFbo)
+
+
+Huwag mag-atubiling magtanong; bawat kontribyutor ay nagsimula sa isang unang PR. Maaaring ikaw ang aming ika-libo!
+
+Kahit na isa kang makaranasang open source na nag-ambag, may mga bagay na maaaring magalit sa iyo. Sa sandaling naisumite mo na ang iyong PR, ang GitHub Actions ay magpapatakbo ng isang linter, kadalasang nakakahanap ng maliliit na isyu sa spacing o alphabetization. Kung nakakuha ka ng berdeng button, handa na ang lahat para sa pagsusuri, ngunit kung hindi, i-click ang "Mga Detalye" sa ilalim ng tseke na nabigong malaman kung ano ang hindi nagustuhan ng linter. Ayusin ang problema at magdagdag ng commit sa iyong PR.
+
+Panghuli, kung hindi ka sigurado na ang resource na gusto mong idagdag ay angkop para sa Free-Programming-Books, basahin ang mga alituntunin sa [CONTRIBUTING](CONTRIBUTING.md).
```

Comment:
Linking with related is safe since https://github.com/EbookFoundation/free-programming-books/pull/6504#issuecomment-955127087 is submitted too.

```suggestion
Panghuli, kung hindi ka sigurado na ang resource na gusto mong idagdag ay angkop para sa Free-Programming-Books, basahin ang mga alituntunin sa [CONTRIBUTING](CONTRIBUTING-fil.md).
```